packages/firebase_auth/firebase_auth/example/test_driver/test_utils.dart

```diff
@@ -1,70 +1,215 @@
 // @dart = 2.9
```

Comment:
just noting that you purged that `// @dart = 2.9` from other files but not from here, stands out

Reply:
I purged duplicates only, was 2 of them in the other file

Reply:
Note to self though; migrate tests to null safety (in follow up PR)